docs/tools/zksync-cli/README.md

```diff
@@ -16,16 +16,22 @@ You can also run this via NPX with `npx zksync-cli@latest [COMMAND]`
 
 ## Commands
 
-- `zksync-cli create [PROJECT_NAME]`: creates a new Hardhat project in a new folder with the given project name. If the project name is not provided, it creates the project in the current folder, although this requires the folder to be empty. The [template project can be found here](https://github.com/matter-labs/zksync-hardhat-template).
-
-- `zksync-cli deposit`: deposits funds from L1 (Goerli testnet) to zkSync Era Testnet. It will ask you to provide: the recipient wallet, the amount in ETH (eg 0.1) and the private key of the wallet you're sending the funds from.
-
-- `zksync-cli withdraw`: withdraws funds from zkSync Era to L1 (Goerli testnet). It will ask you to provide: the recipient wallet, the amount in ETH (eg 0.1) and the private key of the wallet you're sending the funds from.
+- `zksync-cli create [PROJECT_NAME]`: creates a new project in a new folder with the given project name. If the project name is not provided, it creates the project in the current folder, although this requires the folder to be empty.
+- `zksync-cli deposit`: deposits funds from L1 to zkSync Era. It will ask you to provide: the network (localnet, testnet, mainnet), the recipient wallet, the amount in ETH (eg 0.1) and the private key of the wallet you're sending the funds from.
+- `zksync-cli withdraw`: withdraws funds from zkSync Era to L1. It will ask you to provide: the network (localnet, testnet, mainnet), the recipient wallet, the amount in ETH (eg 0.1) and the private key of the wallet you're sending the funds from.
+- `zksync-cli help`: returns the version and information about the different commands.
```

Comment:
Can add `confirm-withdraw` to list of commands

Reply:
fixed in https://github.com/matter-labs/zksync-web-era-docs/pull/633/commits/fcaf423dfe5f1be431438a843e107ce3f3244e6b